server/src/projects/projects.controller.ts

```diff
@@ -31,6 +31,11 @@ import { PROJECT_ATTRS } from './projects.attrs';
       'dcp_packagetype',
       'dcp_visibility',
       'dcp_packageversion',
+      '_dcp_rwcdsform_value',
```

Comment:
Sorry, one change actually. 
Here I think we can just do 
``1
attributes: [
    ...PACKAGE_ATTRS,
]
```
Just like in  Packages.controller.ts

Good catch here to add the attributes!